client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pages/page-preview.tsx

```diff
@@ -54,6 +54,7 @@ const Page = ( { className, style, patterns, transformPatternHtml }: PageProps )
 			{ patterns.map( ( pattern ) => (
 				<PatternRenderer
 					key={ pattern.ID }
+					maxHeight="none"
```

Comment:
Ah, I see the same code here as well

https://github.com/Automattic/wp-calypso/blob/4b8c78a2815459f68e0c418f3da008324f96a21f/client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-large-preview.tsx#L170-L171

Reply:
Yes! IIRC it's only used for pattern rendering in the sidebar.